modules/@apostrophecms/piece-type/ui/apos/components/AposDocsManager.vue

```diff
@@ -72,6 +72,7 @@
             :is-relationship="!!relationshipField"
             :checked-count="checked.length"
             :batch-operations="moduleOptions.batchOperations"
+            :module-name="moduleName"
```

Comment:
AposMediaManager should pass it too. And the doc template library one, in a separate PR.